Formula/raygui.rb

```diff
@@ -0,0 +1,47 @@
+class Raygui < Formula
+  desc "Simple and easy-to-use immediate-mode gui library"
+  homepage "https://github.com/raysan5/raygui"
+  url "https://github.com/raysan5/raygui/archive/refs/tags/3.1.tar.gz"
+  sha256 "4c7d63df44ef1c6a0ddc52b85f3ea32dd480ac764bf2a210cdd6f016a63b5721"
+  license "Zlib"
+
+  depends_on "raylib"
+
+  def install
+    args = [
+      "-shared",
+      "-fpic",
+      "-DRAYGUI_IMPLEMENTATION",
+      "-lraylib",
+      "-lm",
+      "-lpthread",
+      "-ldl",
+    ]
+
+    if OS.mac?
+      args << "-framework" << "OpenGL"
+    else
+      args << "-lGL"
+      args << "-lrt"
+      args << "-lX11"
+    end
+
+    cp "src/raygui.h", "src/raygui.c"
+    system ENV.cc, "-o", shared_library("libraygui"), "src/raygui.c", *args
```

Comment:
Doing this manually isn't great. Is upstream open to accepting the contribution of a `Makefile` so that one can at least do `make PREFIX=$prefix libraygui`?